Added a new example to xDNSServerAddress to clarify setting multiple DNS Servers - fixes #193

PlagueHO opened the pull request:
This PR adds a new example for xDNSServerAddress to show how to set multiple DNS Server Addresses.

Fixes #198

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/powershell/xnetworking/200)
<!-- Reviewable:end -->

tysonjhayes commented on Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1:
```diff
@@ -0,0 +1,24 @@
+    .EXAMPLE
+    Configure primary and secondary DNS Server addresses on the Ethernet adapter
+#>
+configuration Example
```
Given the change I just merged, shouldn't this be capitalized?

johlju replied:
Not until the PR is rebased. Has it been rebased?

PlagueHO commented:
@tysonjhayes - @johlju is correct - I hadn't rebased yet. Done now. Plus the new Example had been added before I made the previous PR to correct the examples.

PlagueHO commented:
@johlju - any chance I could get you to check this one off for me?

johlju commented:
Sure 😄

johlju commented:
Reviewed 1 of 1 files at r2, 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 5 at r1](https://reviewable.io:443/reviews/powershell/xnetworking/200#-Kg1hmVQ0qmfHcxPfeyT-r1-5:-KiKa4bnMrzxFKXHv8n-:b2ooyg4) ([raw file](https://github.com/powershell/xnetworking/blob/ce430235e1d78d18907b653bab534f202a325492/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L5)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Not until the PR is rebased. Has it been rebased?
</blockquote></details>

Need an 'Done' on this comment.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 9 at r3](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiKaMZvCKR-fjKiI0Y3:-KiKaMZvCKR-fjKiI0Y4:bx9f86v) ([raw file](https://github.com/powershell/xnetworking/blob/45ecd018d2a7d6930792331dfafa656d47c0763d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [string[]] $NodeName = 'localhost'
> ```

Add [Parameter()]
Chnage to [System.String[]]
Move  parameter name to a seperate row

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200)*
<!-- Sent from Reviewable.io -->

PlagueHO commented:
@johlju - cool thank you! I did a global replace and corrected the parameter blocks on all the Examples. So unfortunately there are a bunch more files to check :grin:

---

Review status: 1 of 25 files reviewed at latest revision, 2 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 5 at r1](https://reviewable.io:443/reviews/powershell/xnetworking/200#-Kg1hmVQ0qmfHcxPfeyT-r1-5:-KiO0AbCpXfnWRGGpj9Y:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/ce430235e1d78d18907b653bab534f202a325492/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L5)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Need an 'Done' on this comment.
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 9 at r3](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiKaMZvCKR-fjKiI0Y3:-KiO0B2cdfmY2V-1SDnZ:bc5xjdm) ([raw file](https://github.com/powershell/xnetworking/blob/45ecd018d2a7d6930792331dfafa656d47c0763d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
Chnage to [System.String[]]
Move  parameter name to a seperate row
</blockquote></details>

Cool. I fixed all the examples (they all had the same problems).

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200#-:-KiO0Lf6dbkTLrqvIImT:b-ycmn69)*
<!-- Sent from Reviewable.io -->

PlagueHO commented:
Review status: 1 of 25 files reviewed at latest revision, 2 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 9 at r3](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiKaMZvCKR-fjKiI0Y3:-KiO0r2Pa4ExKqDbZqJJ:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/45ecd018d2a7d6930792331dfafa656d47c0763d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L9)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary><blockquote>

Cool. I fixed all the examples (they all had the same problems).
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200)*
<!-- Sent from Reviewable.io -->

johlju commented:
Reviewed 24 of 24 files at r4.
Review status: all files reviewed at latest revision, 25 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPXzMV1K5HaQq__0H0:-KiPXzMWd2p3WGCeOhhr:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/2-SetDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY4fTi9Nwe6Tm4fiF:-KiPY4fTi9Nwe6Tm4fiG:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/2-SetDefaultGateway.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDhcpClient/1-EnableDHCP.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY5WqXDoYX_RFjnqz:-KiPY5WqXDoYX_RFjnr-:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDhcpClient/1-EnableDHCP.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDnsClientGlobalSetting/1-ConfigureSuffixSearchList.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY6ltpUu1D5dhf-tM:-KiPY6ltpUu1D5dhf-tN:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsClientGlobalSetting/1-ConfigureSuffixSearchList.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDnsConnectionSuffix/1-AddSpecificSuffix.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY7ahda8a_t3pXd7e:-KiPY7ahda8a_t3pXd7f:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsConnectionSuffix/1-AddSpecificSuffix.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/1-xDnsServerAddress.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY9dxomsLcI65Un0A:-KiPY9dxomsLcI65Un0B:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/1-xDnsServerAddress.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 5 at r1](https://reviewable.io:443/reviews/powershell/xnetworking/200#-Kg1hmVQ0qmfHcxPfeyT-r1-5:-KiPYEWncWUaFN8zVMZP:bgu2am8) ([raw file](https://github.com/powershell/xnetworking/blob/ce430235e1d78d18907b653bab534f202a325492/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L5)):*
<details><summary><i>Previously, PlagueHO (Daniel Scott-Raynsford) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

That didn't help to resolve the comment, guess @tysonjhayes need to acknowledge this one too. Not a blocking comment anyway...

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYhNDfQ9AgyyPsqZg:-KiPYg15ot69uI1BjR3W:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xFirewall/1-xFirewall_AddFirewallRule.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYisRJjbEVcp40Jwb:-KiPYisRJjbEVcp40Jwc:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/1-xFirewall_AddFirewallRule.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xFirewall/2-xFirewall_AddFirewallRule_AllParameters.ps1, line 11 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYjx3S4ZSUFqOD9Dx:-KiPYjx3S4ZSUFqOD9Dy:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/2-xFirewall_AddFirewallRule_AllParameters.ps1#L11)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xFirewall/3-xFirewall_AddFirewallRuleToExistingGroup.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYkp7IFUARLkhCBqp:-KiPYkp7IFUARLkhCBqq:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/3-xFirewall_AddFirewallRuleToExistingGroup.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xFirewall/4-xFirewall_EnableBuiltInFirewallRule.ps1, line 10 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYlaM3BbO928BFxYV:-KiPYlaM3BbO928BFxYW:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/4-xFirewall_EnableBuiltInFirewallRule.ps1#L10)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xHostsFile/1-AddEntry.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYmUcTgCq9QfEZLQe:-KiPYmUcTgCq9QfEZLQf:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xHostsFile/1-AddEntry.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xHostsFile/2-RemoveEntry.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYnH1duWGYw1j8WWe:-KiPYnH1duWGYw1j8WWf:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xHostsFile/2-RemoveEntry.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xIPAddress/1-AddingStaticIP.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYoCfIf99G7QW8ZRf:-KiPYoCfIf99G7QW8ZRg:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xIPAddress/1-AddingStaticIP.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterBinding/1-DisableIPv6.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYpE3yo_usljP3Y9k:-KiPYpE3yo_usljP3Y9l:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterBinding/1-DisableIPv6.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/1-DisableRDMASettings.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYq5awaRyRtVYvtw2:-KiPYq5awaRyRtVYvtw3:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/1-DisableRDMASettings.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/2-EnableRDMASettings.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYr3m8h5ZzdG2QNwj:-KiPYr3m8h5ZzdG2QNwk:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/2-EnableRDMASettings.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetBIOS/1-DisableNetBIOS.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYrtpy0HwX0R_dJFp:-KiPYrtpy0HwX0R_dJFq:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetBIOS/1-DisableNetBIOS.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetConnectionProfile/1-SetNetConnectionProfile.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYshcMGhBZrupjhhO:-KiPYshcMGhBZrupjhhP:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetConnectionProfile/1-SetNetConnectionProfile.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeam/1-AddNetworkTeam.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYtSdxPoVGpEkhoLj:-KiPYtSdxPoVGpEkhoLk:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeam/1-AddNetworkTeam.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeam/2-RemoveTeam.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYuHtpz70S7g5lkNT:-KiPYuHtpz70S7g5lkNU:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeam/2-RemoveTeam.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/1-AddInterface.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYv6VQ167E8k9wm7D:-KiPYv6VQ167E8k9wm7E:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/1-AddInterface.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/2-RemoveInterface.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYviXFLIXDOE0U-Kt:-KiPYviXFLIXDOE0U-Ku:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/2-RemoveInterface.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---

*[Modules/xNetworking/Examples/Resources/xRoute/1-AddRoute.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYw8WUup6Xwj6Y08-:-KiPYw8WUup6Xwj6Y080:be2tgmi) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xRoute/1-AddRoute.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Add [Parameter()]

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200)*
<!-- Sent from Reviewable.io -->

PlagueHO commented:
@johlju - Sorry about missing the `[Parameter()]` - very careless of me. Thanks for reviewing!

---

Review status: all files reviewed at latest revision, 25 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPXzMV1K5HaQq__0H0:-KiTgeahl6MO51v_TF2D:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/2-SetDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY4fTi9Nwe6Tm4fiF:-KiTgfY5o1VW4r0O_6HW:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/2-SetDefaultGateway.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDhcpClient/1-EnableDHCP.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY5WqXDoYX_RFjnqz:-KiTggCYABxS7jbzUWm0:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDhcpClient/1-EnableDHCP.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDnsClientGlobalSetting/1-ConfigureSuffixSearchList.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY6ltpUu1D5dhf-tM:-KiTggvDB0jnXXVP64IG:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsClientGlobalSetting/1-ConfigureSuffixSearchList.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDnsConnectionSuffix/1-AddSpecificSuffix.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY7ahda8a_t3pXd7e:-KiTghqatEjlQv4HyeLH:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsConnectionSuffix/1-AddSpecificSuffix.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/1-xDnsServerAddress.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPY9dxomsLcI65Un0A:-KiTgj-l0sjY2NHAnMje:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/1-xDnsServerAddress.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 5 at r1](https://reviewable.io:443/reviews/powershell/xnetworking/200#-Kg1hmVQ0qmfHcxPfeyT-r1-5:-KiTgnWNw-eqweWOJEaR:b-j2fohf) ([raw file](https://github.com/powershell/xnetworking/blob/ce430235e1d78d18907b653bab534f202a325492/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L5)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

That didn't help to resolve the comment, guess @tysonjhayes need to acknowledge this one too. Not a blocking comment anyway...
</blockquote></details>

I think you're right.

---

*[Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYhNDfQ9AgyyPsqZg:-KiTgwECuokuL_PpMByz:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDnsServerAddress/2-xDnsServerAddress_PrimaryAndSecondary.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xFirewall/1-xFirewall_AddFirewallRule.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYisRJjbEVcp40Jwb:-KiTgwy7nP8zLarQVRO8:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/1-xFirewall_AddFirewallRule.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xFirewall/2-xFirewall_AddFirewallRule_AllParameters.ps1, line 11 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYjx3S4ZSUFqOD9Dx:-KiTgxoQ2CZJ8EiGau1x:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/2-xFirewall_AddFirewallRule_AllParameters.ps1#L11)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xFirewall/3-xFirewall_AddFirewallRuleToExistingGroup.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYkp7IFUARLkhCBqp:-KiTgyXpptM35q9di9ci:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/3-xFirewall_AddFirewallRuleToExistingGroup.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xFirewall/4-xFirewall_EnableBuiltInFirewallRule.ps1, line 10 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYlaM3BbO928BFxYV:-KiTgzJ8K-rvSTpf6sp2:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xFirewall/4-xFirewall_EnableBuiltInFirewallRule.ps1#L10)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xHostsFile/1-AddEntry.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYmUcTgCq9QfEZLQe:-KiTh--WbCyauDCM4sz0:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xHostsFile/1-AddEntry.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xHostsFile/2-RemoveEntry.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYnH1duWGYw1j8WWe:-KiTh-h__8tc6A_Eh2bJ:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xHostsFile/2-RemoveEntry.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xIPAddress/1-AddingStaticIP.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYoCfIf99G7QW8ZRf:-KiTh0RJ5shn7GeOonfv:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xIPAddress/1-AddingStaticIP.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterBinding/1-DisableIPv6.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYpE3yo_usljP3Y9k:-KiTh19E609DQB7LGdlH:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterBinding/1-DisableIPv6.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/1-DisableRDMASettings.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYq5awaRyRtVYvtw2:-KiTh1luWUMg8HasILSO:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/1-DisableRDMASettings.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/2-EnableRDMASettings.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYr3m8h5ZzdG2QNwj:-KiTh2QkXqZZ8qS51ClR:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetAdapterRDMA/2-EnableRDMASettings.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetBIOS/1-DisableNetBIOS.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYrtpy0HwX0R_dJFp:-KiTh3Hfwg640-V_dUSg:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetBIOS/1-DisableNetBIOS.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetConnectionProfile/1-SetNetConnectionProfile.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYshcMGhBZrupjhhO:-KiTh4-fbGiLc0ETz8SI:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetConnectionProfile/1-SetNetConnectionProfile.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeam/1-AddNetworkTeam.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYtSdxPoVGpEkhoLj:-KiTh4jkf8Hs0agUHosg:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeam/1-AddNetworkTeam.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeam/2-RemoveTeam.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYuHtpz70S7g5lkNT:-KiTh5mHv5mlQ0OrzrSG:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeam/2-RemoveTeam.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/1-AddInterface.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYv6VQ167E8k9wm7D:-KiTh6PQZmNNnRQlYXLf:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/1-AddInterface.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/2-RemoveInterface.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYviXFLIXDOE0U-Kt:-KiTh79obO6d67WhioYl:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xNetworkTeamInterface/2-RemoveInterface.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---

*[Modules/xNetworking/Examples/Resources/xRoute/1-AddRoute.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiPYw8WUup6Xwj6Y08-:-KiTh8-uEllDDXa0OPFb:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xRoute/1-AddRoute.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Add [Parameter()]
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200#-:-KiTh9H_oGgx6EtUajVd:bvfsk6a)*
<!-- Sent from Reviewable.io -->

johlju commented:
No worries, even the sun has spots :wink:  Happy to review :smile: 

But, my friend... I think you forgot to push the changes :laughing: 

---

Review status: all files reviewed at latest revision, 25 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiVdmReE-AhYihjnuTn:-KiVdmReE-AhYihjnuTo:bg90i9x) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1#L9)):*
> ```PowerShell
>     param
>     (
>         [System.String[]]
> ```

Still no [Parameter()] anywhere :smile: 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200#-:-KiVdBAo-htoft40rAQa:b-21bwtj)*
<!-- Sent from Reviewable.io -->

PlagueHO commented:
@johlju - you are right! I totally forgot to push the change. Making many mistakes these last couple of days. Sorry again.

---

Review status: 0 of 25 files reviewed at latest revision, 25 unresolved discussions.

---

*[Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1, line 9 at r4](https://reviewable.io:443/reviews/powershell/xnetworking/200#-KiVdmReE-AhYihjnuTn:-KiWcF6dq6Hj6Ipw-fDu:b-896fix) ([raw file](https://github.com/powershell/xnetworking/blob/8bd24a99a191b711edd56cb0bfd558d6eada8d3d/Modules/xNetworking/Examples/Resources/xDefaultGatewayAddress/1-RemoveDefaultGateway.ps1#L9)):*
<details><summary><i>Previously, johlju (Johan Ljunggren) wrote…</i></summary><blockquote>

Still no [Parameter()] anywhere :smile: 
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200#-:-KiWc4-79b1TVi1FoucH:b-9yvc1c)*
<!-- Sent from Reviewable.io -->

johlju commented:
<img class="emoji" title=":LGTM:" alt=":LGTM:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 25 of 25 files at r5.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/powershell/xnetworking/200#-:-Ki_qm_ITCWI3QtQFni2:bmxilld)*
<!-- Sent from Reviewable.io -->

johlju commented:
No worries! 😁 It's okay to make mistakes 😄 

Only Tyson's comment left that we can't acknowledge. But I think you can merge anyway. It's is not a blocking comment.

tysonjhayes commented:
heh... sorry ya'll been busy with work related things I've spaced on paying attention to this. Signed off. Looks great. Thanks everyone!

PlagueHO commented:
Hehe - thanks @tysonjhayes - you've got a good excuse :grin:
